packages/sdk/src/browser-wallets/magiceden/index.ts

```diff
@@ -19,33 +20,49 @@ export interface MagicEdenWindow extends Window {
   BitcoinProvider?: MagicEdenBitcoinProvider;
 }
 
+export interface MagicEdenWallet extends Wallet {
+  name: "Magic Eden";
+  features: {
+    "sats-connect:": {
+      provider: MagicEdenBitcoinProvider;
+    };
+  };
+}
+
+async function getMagicEdenWalletProvider(): Promise<BitcoinProvider> {
+  const meWallet = getWallets()
+    .get()
+    .find((wallet) => wallet.name === "Magic Eden");
+
+  if (!meWallet) {
+    throw new BrowserWalletNotInstalledError("Magic Eden not installed.");
+  }
+
+  const magicEdenWalletProvider = (meWallet as MagicEdenWallet).features[
+    "sats-connect:"
+  ]!.provider;
```

Comment:
can we remove the non-null assertion here since the type assumes that provider always exists anyway?

Reply:
Removed.